chain-impl-mockchain/src/rewards.rs

```diff
@@ -249,6 +329,8 @@ mod tests {
         );
     }
 
+    // would be good to add some additional reward calculation tests here, including scaling etc. KH
```

Comment:
superfluous comment

Reply:
A placeholder/indicator that some more work could be done.  We'll remove it, of course.